src/main/telemetry/jetiexbus.c

```diff
@@ -85,15 +81,19 @@ enum {
 };
 
 enum exDataType_e {
-    EX_TYPE_6b   = 0, // int6_t  Data type 6b (-31 ¸31)
-    EX_TYPE_14b  = 1, // int14_t Data type 14b (-8191 ¸8191)
-    EX_TYPE_22b  = 4, // int22_t Data type 22b (-2097151 ¸2097151)
-    EX_TYPE_DT   = 5, // int22_t Special data type – time and date
-    EX_TYPE_30b  = 8, // int30_t Data type 30b (-536870911 ¸536870911)
-    EX_TYPE_GPS  = 9  // int30_t Special data type – GPS coordinates:  lo/hi minute - lo/hi degree.
+    EX_TYPE_NONE = 0,
+    EX_TYPE_6b   = 0,                // int6_t  Data type 6b (-31 ¸31)
+    EX_TYPE_14b  = 1,                // int14_t Data type 14b (-8191 ¸8191)
+    EX_TYPE_22b  = 4,                // int22_t Data type 22b (-2097151 ¸2097151)
+    EX_TYPE_DT   = 5,                // int22_t Special data type – time and date
+    EX_TYPE_30b  = 8,                // int30_t Data type 30b (-536870911 ¸536870911)
+    EX_TYPE_GPS  = 9,                // int30_t Special data type – GPS coordinates:  lo/hi minute - lo/hi degree.
+    EX_TYPE_SENSOR_DISABLED = 240,
```

Comment:
ENABLED/DISABLED is used for parameter, not type

Reply:
Is removed. A bitmask was used as suggested.